CMakeLists.txt

```
@@ -113,6 +113,10 @@ configure_file(templates/setup.sh.in ${CMAKE_CURRENT_BINARY_DIR}/setup.sh @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
     DESTINATION ${CMAKE_INSTALL_PREFIX}
 )
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
+    DESTINATION ${CMAKE_INSTALL_BINDIR}
```

If we are talking FHS, then this would be the place
```suggestion
    DESTINATION ${CMAKE_INSTALL_LIBEXEC}
```
In practice, bindir is fine too, of course.

---

Right so. I'm worried people aren't going to find libexec very convenient...